<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreWidgets</class>
 <widget class="QDialog" name="MoreWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background:rgba(255, 255, 255, 0)}

    QPushButton{
        padding: 3px 10px 3px 10px;
        color: rgba(255, 255, 255, 195);
        border: 1px solid rgba(255, 255, 255, 50);
        border-radius:2px;
        background-color: rgba(0, 0, 0, 50);
    }

    QPushButton::flat:hover{
        padding: 3px 10px 3px 10px;
        color: rgba(255, 255, 255, 195);
        border: 1px solid rgba(255, 255, 255, 80);
        border-radius:2px;
        background-color: rgba(0, 0, 0, 80);
     }

    QPushButton::flat{
        padding: 3px 10px 3px 10px;
        color: rgba(255, 255, 255, 195);
        border: 1px solid rgba(255, 255, 255, 50);
        border-radius:2px;
        background-color: rgba(0, 0, 0, 50);
    }

    QPushButton::flat:pressed{
        padding: 3px 10px 3px 10px;
        color: rgba(255, 255, 255, 195);
        border: 1px solid rgba(255, 255, 255, 30);
        border-radius:2px;
        background-color: rgba(0, 0, 0, 100);
    }

  #randomPackageList {
        color: rgb(255,255,255,195);
        border-bottom-left-radius:2px;
        border-bottom-left-radius:2px;
        background-color: rgba(255, 255, 255, 20);
       /*background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:0, y2:1,
            stop:0 rgba(27, 27, 27, 100),
            stop:0.510753 rgba(9, 9, 9, 100),
            stop:1 rgba(0, 0, 0, 100));*/
        show-decoration-selected: 1; /* make the selection span the entire width of the view */
    }

    #randomPackageList::item{
        border-radius: 2px;
        color: rgb(255,255,255,195);
        border:1px solid  rgba(255, 255, 255, 5);
        padding: 5px;
        background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0,
            stop:0 rgba(0, 0, 0, 10));
    }

    #randomPackageList::item:selected {
        border:1px solid  rgba(255, 255, 255, 255);
	    background-color: rgb(0, 0, 0);
    }

    #randomPackageList::item:hover {
        border:1px solid  rgba(255, 255, 255, 150);
	    background-color: rgba(0, 0, 0, 150);
    }
/* #QScrollArea */

    QScrollArea::corner {
            background-color: rgba(255, 255, 255, 0);
            border: 0px;
    }

    QScrollBar:vertical {
            width: 13px;
            margin: 3px 3px 3px 3px;
            background-color: rgba(255, 255, 255, 0);
    }

    QScrollBar::handle:vertical {
            min-height: 16px;
            background-color: rgba(0,0,0, 15);
            border: 1px solid rgb(255, 255, 255, 25);
            border-radius: 3px;
            padding:3px;
    }

    QScrollBar::handle:vertical:hover {
            min-height: 16px;
            background-color: rgba(255,255,255, 25);
            border-radius: 3px;
    }

    QScrollBar::add-line:vertical {
            border: 0px;
            height: 0px;
            subcontrol-position: bottom;
            subcontrol-origin: margin;
            background: transparent;
    }

    QScrollBar::sub-line:vertical {
            border: 0px;
            height: 0px;
            subcontrol-position: top;
            subcontrol-origin: margin;
            background: transparent;
    }

    QScrollBar:horizontal {
            height: 13px;
            margin: 3px 3px 3px 3px;
            background: rgba(255, 255, 255, 0);
    }

    QScrollBar::handle:horizontal {
            min-width: 16px;
            background-color: rgba(0,0,0, 15);
            border: 1px solid rgb(255, 255, 255, 25);
            border-radius: 3px;
    }

    QScrollBar::handle:horizontal:hover {
            min-width: 16px;
            background-color: rgba(255,255,255, 25);
            border-radius: 3px;
    }

    QScrollBar::add-line:horizontal {
            border: 0px;
            background: transparent;
            width: 0px;
            subcontrol-position: right;
            subcontrol-origin: margin;
    }

    QScrollBar::sub-line:horizontal {
            border: none;
            width: 0px;
            subcontrol-position: left;
            subcontrol-origin: margin;
    }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>-1</number>
   </property>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="mwMainlayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <item>
            <widget class="QPushButton" name="updateSlideButton">
             <property name="styleSheet">
              <string notr="true">color: white;</string>
             </property>
             <property name="text">
              <string>Update Slide</string>
             </property>
             <property name="icon">
              <iconset resource="../data/data.qrc">
               <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="shortcut">
              <string>Esc</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWebView" name="webView">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>280</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>690</width>
               <height>280</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(255, 255, 255, 0)</string>
             </property>
             <property name="url">
              <url>
               <string>file:///media/Arsiv2/packagemanager/Onur/index.html</string>
              </url>
             </property>
             <property name="zoomFactor">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgba(255, 255, 255, 150);
background-color: rgba(255, 255, 255, 20);
border-top:1px solid rgba(255, 255, 255, 30);
border-top-left-radius:2px;
border-top-right-radius:2px;</string>
               </property>
               <property name="text">
                <string>Popular packages</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <property name="margin">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="randomPackageList">
               <property name="minimumSize">
                <size>
                 <width>700</width>
                 <height>210</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>700</width>
                 <height>210</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>true</bool>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::NoDragDrop</enum>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
               <property name="verticalScrollMode">
                <enum>QAbstractItemView::ScrollPerItem</enum>
               </property>
               <property name="movement">
                <enum>QListView::Static</enum>
               </property>
               <property name="isWrapping" stdset="0">
                <bool>true</bool>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="gridSize">
                <size>
                 <width>95</width>
                 <height>95</height>
                </size>
               </property>
               <property name="viewMode">
                <enum>QListView::IconMode</enum>
               </property>
               <property name="batchSize">
                <number>100</number>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/data.qrc"/>
 </resources>
 <connections/>
</ui>
